api/types.proto

```diff
@@ -277,6 +277,50 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum FailureAction {
+		PAUSE = 0;
+		CONTINUE = 1;
+		// TODO(aaronl): Add ROLLBACK as a supported failure mode.
```

Comment:
This will probably have a time component for the rollback. For example, we'd saying "rollback after two minutes" if the update fails.